src/collect-offline-maps.rst

```diff
@@ -0,0 +1,44 @@
+.. spelling:: 
+  basemap
+  geospatial
+  heatmaps
+  Mapbox
+  pbf
+
+Using Offline Maps
+====================
+
+Collect's :ref:`geoshape`, :ref:`geotrace` and :ref:`geopoint` question types can be configured to display different maps. To use online map tiles, set the mapping engine and basemap from the :ref:`mapping settings <mapping-settings>`. Users will need to be online to load questions with maps.
```

Comment:
>  set the mapping engine and basemap from the :ref:`mapping settings <mapping-settings>`

I realize this isn't documented anywhere, but the emergent preferred style for this is to link the action:

```
:ref:`set the mapping engine and basemap <mapping-settings>` in :ref:`interface-settings`

Reply:
Done.